src/main/java/taxiservice/Application.java

```diff
@@ -0,0 +1,52 @@
+package taxiservice;
+
+import taxiservice.lib.Injector;
+import taxiservice.model.Manufacturer;
+import taxiservice.service.CarService;
+import taxiservice.service.DriverService;
+import taxiservice.service.ManufacturerService;
+
+public class Application {
+    private static final Injector injector = Injector.getInstance("taxiservice");
+
+    public static void main(String[] args) {
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        CarService carService = (CarService)
+                injector.getInstance(CarService.class);
+
+        Manufacturer manufacturer1 = new Manufacturer("Toyota", "Japan");
```

Comment:
```suggestion
        Manufacturer toyotaManufacturer = new Manufacturer("Toyota", "Japan");
```
Let's use meaningful names